syntax = "proto3";

package myapp;

service RouteService {
    rpc upload(stream Route) returns (Route) {}
    rpc query(QueryRequest) returns (Route) {}
}

message Route {
    string id = 1;
    bytes payload = 2;
}

message QueryRequest {
    string issue_date = 1;
}